jwst/outlier_detection/ifu.py

```diff
@@ -73,48 +94,70 @@ def detect_outliers(
 
     # check if kernel size is an odd value
     if kern_size[0] % 2 == 0:
-        log.info("X kernel size is given as an even number. This value must be an odd number. Increasing number by 1")
+        log.info(
+            "X kernel size is given as an even number. This value must be an odd number. "
+            "Increasing number by 1"
+        )
         kern_size[0] = kern_size[0] + 1
-        log.info("New x kernel size is {}: ".format(kern_size[0]))
+        log.info("New x kernel size is {kern_size[0}: ")
```

Comment:
```suggestion
        log.info(f"New x kernel size is {kern_size[0]}: ")
```
Was this a manual update or is there a buggy rule(s)?

Reply:
That was a bad manual update, I probably should have self-reviewed this more carefully. Thanks for catching, will fix